sunpy/net/helio/hec.py

```diff
@@ -185,3 +284,6 @@ def select_table(self):
                 return table_list[table_no - 1]
             else:
                 print(f"Input must be an integer between 1 and {len(table_list)}")
+
+    def fetch(self, *args, **kwargs):
+        return NotImplemented
```

Comment:
Should this not just return whatever it put into it?

Reply:
It won't be called ever. You mean return `HECResponse`? It may create confusion. We will discuss more on it.